packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.vue

```diff
@@ -24,6 +24,7 @@
     <component
       :is="injections.components.SfInput"
       type="number"
+      :name="'quantitySelector' + $options.uniqueKey()"
```

Comment:
suggestion: I would move it to a computed property :) fe. inside uniqueKey so it returns 'quantitySelector' + key